workloads/kube-burner/env.sh

```diff
@@ -1,24 +1,41 @@
+#!/usr/bin/env bash
 
-# Common
+# Benchark-operator
 export OPERATOR_REPO=${OPERATOR_REPO:-https://github.com/cloud-bulldozer/benchmark-operator.git}
 export OPERATOR_BRANCH=${OPERATOR_BRANCH:-master}
-export QPS=${QPS:-20}
-export BURST=${BURST:-20}
-export POD_READY_TIMEOUT=${POD_READY_TIMEOUT:-1200}
-export WORKLOAD_NODE=${WORKLOAD_NODE:-'{"node-role.kubernetes.io/worker": ""}'}
-export CERBERUS_URL=${CERBERUS_URL}
-export STEP_SIZE=${STEP_SIZE:-30s}
+export POD_READY_TIMEOUT=${POD_READY_TIMEOUT:-180}
+
+# Indexing variables
+export INDEXING=${INDEXING:-true}
 export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
 export ES_INDEX=${ES_INDEX:-ripsaw-kube-burner}
-export PROM_URL=${PROM_URL:-https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091}
+export STEP_SIZE=${STEP_SIZE:-30s}
 export METADATA_COLLECTION=${METADATA_COLLECTION:-true}
-export JOB_TIMEOUT=${JOB_TIMEOUT:-14400}
-export LOG_STREAMING=${LOG_STREAMING:-true}
+export PROM_URL=${PROM_URL:-https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091}
+
+# Kube-burner vars
+export QPS=${QPS:-20}
+export BURST=${BURST:-20}
+export MAX_WAIT_TIMEOUT=${MAX_WAIT_TIMEOUT:-1h}
 export CLEANUP=${CLEANUP:-true}
-export CLEANUP_WHEN_FINISH=${CLEANUP_WHEN_FINISH:-false}
-export LOG_LEVEL=${LOG_LEVEL:-info}
+export POD_NODE_SELECTOR=${POD_NODE_SELECTOR:-'{node-role.kubernetes.io/worker: }'}
+export WAIT_WHEN_FINISHED=true
+export POD_WAIT=${POD_WAIT:-false}
+export WAIT_FOR=${WAIT_FOR:-[]}
+export VERIFY_OBJECTS=${VERIFY_OBJECTS:-true}
+export ERROR_ON_VERIFY=${ERROR_ON_VERIFY:-true}
+export TEST_JOB_ITERATIONS=${JOB_ITERATIONS:-1000}
```

Comment:
This should be JOB_ITERATIONS not Test_Job... since its in the env.sh file

Reply:
I have removed it from env.sh as the current implementation. Since JOB_ITERATIONS is only used in cluster-density I added it there.